docs/setup/install.md

```diff
@@ -0,0 +1,29 @@
+# Installation
+
+## Before you begin
+
+Make sure the following conditions are met:
+
+- A Kubernetes cluster is running.
+- The kubectl command-line tool has communication with your cluster.
+
+Kueue **Requires Kubernetes 1.22 or newer**. Learn how to [install the Kubernetes tools](https://kubernetes.io/docs/tasks/tools/).
+Kubernetes 1.21 can be used if the SuspendJob [feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) is enabled
```

Comment:
```suggestion
Note: You can use Kubernetes 1.21 if the SuspendJob [feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) is enabled.
```

Sounds more like a note because it could be a side use case.